src/vue/modules/issuances-explorer/store/index.spec.js

```diff
@@ -0,0 +1,95 @@
+import { mutations, getters, actions } from './index'
+import { types } from './types'
+
+import { Issuance } from '../wrappers/issuance'
+
+import { Sdk } from '@/sdk'
+import { OP_TYPES } from '@tokend/js-sdk'
+
+describe('issuances-explorer.module', () => {
+  const accountId = 'GDIU5OQPAFPNBP75FQKMJTWSUKHTQTBTHXZWIZQR4DG4QRVJFPML6TTJ'
+  const issuances = [{
+    id: '1',
+    amount: '100.000000',
+  }, {
+    id: '2',
+    amount: '50.000000',
+  }]
+
+  describe('mutations', () => {
+    let state
+
+    beforeEach(() => {
+      state = {
+        accountId: '',
+        issuances: [],
+      }
+    })
+
+    it('SET_ACCOUNT_ID should properly modify state', () => {
+      mutations[types.SET_ACCOUNT_ID](state, accountId)
+
+      expect(state).to.deep.equal({
+        accountId: accountId,
+        issuances: [],
+      })
+    })
+
+    it('SET_ISSUANCES should properly modify state', () => {
+      mutations[types.SET_ISSUANCES](state, issuances)
+
+      expect(state).to.deep.equal({
+        accountId: '',
+        issuances,
+      })
+    })
+
+    it('CONCAT_ISSUANCES should properly modify state', () => {
+      mutations[types.SET_ISSUANCES](state, issuances)
```

Comment:
```
const state = {
  issuances: [1,2,3]
}

mutation(state, [4,5,6])

expect(state).to.deep.equal([1,2,3,4,5,6])
```

Reply:
every call inside the test case is redundant, except this is the call of the function you're testing

Reply:
Updated